rl_agent/rl_env.py

```diff
@@ -32,13 +36,26 @@ def reset(self, seed=None, options={}):
             # There is some programs that has unsupported loop levels , acces matrices , ...
             # These programs are not supported yet so the embedded_tensor will be None
             # program = random.choice(self.tiramisu_api.programs)
-            self.program_name , data = ray.get(self.dataset_actor.get_next_function.remote())
+            try:
```

Comment:
The goal of having tiramisu API is to leave all treatments of tiramisu outside the reinforcement learning environment , try catching the errors related to tirmaisu env should not happen in the rl env , the rl environrment should not care if  a program is supported or not , this should happen in the api. What i suggest is the following :

- Don't create tiramisu prog in the rl env 
- Pass the data needed to create tiramisu prog into the set_program() function 
- treat everything in that function and return None if the program is unsupported 
- The Rl env will only receive a none value if a program is unsupported